src/box/card.css

```diff
@@ -0,0 +1,83 @@
+.card {
+  color: rgba(6, 11, 53, 1);
+  background-color: rgba(255, 255, 255, 1);
+
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px;
+
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+}
+
+.card__imac-foto {
+  height: 134px;
+  width: 160px;
+}
+
+.card__foto-container {
+  display: flex;
+  padding: 0 0 40px;
```

Comment:
Try to use margin instead paddings, because we need to use margin for external indents, and padding for internal